systemdev/ubp-5.1.rst

```diff
@@ -1,10 +1,12 @@
-Legacy porting
-==============
+hammerhead, bacon and FP2
```

Comment:
This is obvious, but not so for someone who is starting from scratch.. I'd suggest being more explicit here:

"Nexus 5 (hammerhead), OnePlus One (bacon) and Fairphone 2 (FP2)"

Reply:
Yeah ... the problem is that the title of the page is also used in the navigation on the left side ... and then it gets really long and looks ugly there .... not sure how to best solve this